cirq/protocols/approximate_equality_protocol_test.py

```diff
@@ -152,6 +153,23 @@ def test_approx_eq_list():
     assert not cirq.approx_eq([1.1, 1.2, 1.3], [1, 1, 1], atol=0.2)
 
 
+def test_approx_eq_symbol():
+    q = cirq.GridQubit(0, 0)
+    s = sympy.Symbol("s")
+    t = sympy.Symbol("t")
+
+    no_symbol_1 = cirq.Circuit(cirq.rz(1.515)(q))
```

Comment:
these should be covered elsewhere, no need to repeat in this test

Reply:
Done